app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitValidator.java

```diff
@@ -48,6 +52,11 @@ public boolean isHabitValid(String habitTitle,
             return false;
         }
 
+        if (tracker.getDays().isEmpty()){
+            showToastMessage("Please select what days of the week.");
+            return false;
+        }
```

Comment:
This is the message that displays when a habit has no days selected. This can be seen when editing or adding a habit.